src/ttf.cr

```diff
@@ -139,35 +139,42 @@ module SDL
 
       # Returns the `{width, height}` of *text* if rendered with the current font.
       def size_of(text : String, ascii = false)
+        width : LibC::Int = -1
+        height : LibC::Int = -1
```

Comment:
You can left these uninitialized:
```suggestion
        width = uninitialized LibC::Int
        height = uninitialized LibC::Int
```

The same applies for the other methods.

Reply:
I prefer to initialize with an invalid value just in case, but I'll make the change if you prefer that.

Reply:
It's safe because we either always set them or raise, and thus don't use them.